include/boost/beast/http/impl/file_body_win32.hpp

```diff
@@ -78,6 +78,12 @@ struct basic_file_body<file_win32>
         value_type(value_type&& other) = default;
         value_type& operator=(value_type&& other) = default;
 
+        /// Accessor to contained File instance
```

Comment:
Should read
```
/// Return the file
```

Reply:
I think that one does not generate docs and has no other doc comments, so I just removed it